adapters/from_sql.py

```diff
@@ -150,25 +153,35 @@ def get_task_info(dag_id, execution_date, airflow_db):
       most_recent_task = task
     elif task.status == STATE_FROM_STRING.get('running'):
       state = task.status
+      most_recent_task = task
     elif task.status > state:
       state = task.status
+    logging.debug(task.status)
+  print '????????'
+  print state
   return task_ids, most_recent_task, state
 
 
 def read_xcom_vars(xcom_data):
   """Reads the xcom data to get the dict of vars and the green build SHA."""
   # occasionally, the xcom data doesn't exist yet. This is a workaround
-  xcom_named_tuple = namedtuple('Row', ['xcom_dict', 'green_sha'])
+
```

Comment:
We were having an issue where if a release failed get_git_commit it wouldn't have a SHA so it would never be turned into a release object and wouldn't show up on the UI. I didn't know the best way to access the tuple in the case that there was only the xcom_dict and no green_sha field.